src/components/Table/TableRow.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import React from 'react';
+import theme from '../../theme';
+import { COLOR, COMMON, TYPOGRAPHY } from '../../utils/constants';
+import TableHeadContext from './TableHeadContext';
+
+
+export const TableRowWrapper = styled.tr`
+    height: 52px;
+    ${(props) => props.isHeadRow && css`
+        height: 56px;
+    `}
```

Comment:
didn't know about this css``, cool stuff

if you want to get rid of a few lines you can change this to
```Javascript
    height: ${(props) => (props.isHeadRow ? '56px' : '52px')}
```